goaviatrix/client_proxy_config.go

```diff
@@ -0,0 +1,107 @@
+package goaviatrix
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/url"
+	"strings"
+)
+
+type ClientProxyConfig struct {
+	HttpProxy          string `form:"http_proxy,omitempty" json:"http_proxy,omitempty"`
+	HttpsProxy         string `form:"https_proxy,omitempty" json:"https_proxy,omitempty"`
+	ProxyCaCertificate string `form:"server_ca_cert,omitempty" json:"server_ca_cert,omitempty"`
```

Comment:
The JSON omitempty tag is also unnecessary since this struct is only used for decoding